client/common/network.go

```diff
@@ -36,101 +184,144 @@ func LoadNetwork(name string, node string) Network {
+			ExplorerGrpcEndpoint:    "tcp://devnet.api.injective.dev:9911",
+			ChainId:                 "injective-777",
+			Fee_denom:               "inj",
+			Name:                    "devnet",
+			chainCookieAssistant:    &DisabledCookieAssistant{},
+			exchangeCookieAssistant: &DisabledCookieAssistant{},
+			explorerCookieAssistant: &DisabledCookieAssistant{},
 		}
 	case "testnet":
-		validNodes := []string{"sentry0", "sentry1", "k8s"}
+		validNodes := []string{"lb", "lb_k8s", "sentry", "sentry0", "sentry1"}
 		if !contains(validNodes, node) {
 			panic(fmt.Sprintf("invalid node %s for %s", node, name))
 		}
 
 		var lcdEndpoint, tmEndpoint, chainGrpcEndpoint, exchangeGrpcEndpoint, explorerGrpcEndpoint string
 		var chainTlsCert, exchangeTlsCert, explorerTlsCert credentials.TransportCredentials
-		if node == "k8s" {
+		var chainCookieAssistant, exchangeCookieAssistant, explorerCookieAssistant CookieAssistant
+		if node == "lb" {
+			lcdEndpoint = "https://testnet.sentry.lcd.injective.network:443"
+			tmEndpoint = "https://testnet.sentry.tm.injective.network:443"
+			chainGrpcEndpoint = "testnet.sentry.chain.grpc.injective.network:443"
+			exchangeGrpcEndpoint = "testnet.sentry.exchange.grpc.injective.network:443"
+			explorerGrpcEndpoint = "testnet.sentry.explorer.grpc.injective.network:443"
+			chainTlsCert = credentials.NewServerTLSFromCert(&tls.Certificate{})
+			exchangeTlsCert = credentials.NewServerTLSFromCert(&tls.Certificate{})
+			explorerTlsCert = credentials.NewServerTLSFromCert(&tls.Certificate{})
+			chainCookieAssistant = &BareMetalLoadBalancedCookieAssistant{}
+			exchangeCookieAssistant = &BareMetalLoadBalancedCookieAssistant{}
+			explorerCookieAssistant = &BareMetalLoadBalancedCookieAssistant{}
+		} else if node == "lb_k8s" {
 			certPath := getFileAbsPath("../cert/testnet.crt")
 			lcdEndpoint = "https://k8s.testnet.lcd.injective.network"
 			tmEndpoint = "https://k8s.testnet.tm.injective.network:443"
 			chainGrpcEndpoint = "tcp://k8s.testnet.chain.grpc.injective.network:443"
-			chainTlsCert = LoadTlsCert(certPath, chainGrpcEndpoint)
 			exchangeGrpcEndpoint = "tcp://k8s.testnet.exchange.grpc.injective.network:443"
 			explorerGrpcEndpoint = "tcp://k8s.testnet.explorer.grpc.injective.network:443"
+			chainTlsCert = LoadTlsCert(certPath, chainGrpcEndpoint)
 			exchangeTlsCert = LoadTlsCert(certPath, exchangeGrpcEndpoint)
 			explorerTlsCert = LoadTlsCert(certPath, explorerGrpcEndpoint)
+			chainAssistant := TestnetKubernetesCookieAssistant()
+			chainCookieAssistant = &chainAssistant
+			exchangeAssistant := TestnetKubernetesCookieAssistant()
+			exchangeCookieAssistant = &exchangeAssistant
+			explorerAssistant := TestnetKubernetesCookieAssistant()
+			explorerCookieAssistant = &explorerAssistant
+		} else if node == "sentry" {
+			lcdEndpoint = "https://testnet.lcd.injective.network"
+			tmEndpoint = "https://testnet.tm.injective.network"
+			chainGrpcEndpoint = "testnet.chain.grpc.injective.network"
```

Comment:
please add missing port here (443)

Reply:
Hi @vinhphuctadang, thank you for the comment.
This is the exact same configuration we have in the Python SDK (https://github.com/InjectiveLabs/sdk-python/blob/c247dbe2142b2c932ebd47acb8e7fb0796376d7d/pyinjective/core/network.py#L224) that is working without issues?
I think it would be better to keep the configuration consistent between SDKs

Reply:
You were right @vinhphuctadang. In Python the connection works without the port number, but in Go it fails. I have updated the configuration in Go SDK to include the port number, and I will change the code in Python SDK to synchronize the endpoints